core/wire/noise_test.go

```diff
@@ -38,18 +43,19 @@ func TestNyquistPqNoiseParams2(t *testing.T) {
 	seecGenRand, err := seec.GenKeyPRPAES(rand.Reader, 256)
 	require.NoError(t, err, "seec.GenKeyPRPAES")
 
-	//protocol2, err := nyquist.NewProtocol("Noise_pqXX_Kyber768X25519_ChaChaPoly_BLAKE2s")
-	//require.NoError(t, err)
-
 	protocol := &nyquist.Protocol{
 		Pattern: pattern.PqXX,
-		KEM:     kem.Kyber768X25519,
-		Cipher:  cipher.ChaChaPoly,
-		Hash:    hash.BLAKE2s,
+		KEM: kem.FromKEM(
+			kemhybrid.New(
+				"Kyber768-X25519",
+				adapter.FromNIKE(ecdh.NewEcdhNike(rand.Reader)),
+				kyber768.Scheme(),
+			),
+		),
+		Cipher: cipher.ChaChaPoly,
+		Hash:   hash.BLAKE2s,
```

Comment:
Do we have blake2b? I think it'd be interesting to see a benchmark between the two; as I understand it blake2s is the microcontroller-optimized algorithm, while we are probably targeting 64-bit machines here?
If we're changing all the other things we might as well deal with this now?

Reply:
Yeah I think we mostly care about 64-bit machines. blake2b is probably a fine choice.

Reply:
lol okay please limit the scope of this code review to the code that i actually wrote in the pull request.